types/Webhooks.d.ts

```diff
@@ -0,0 +1,64 @@
+declare module 'stripe' {
+  namespace Stripe {
+    export class Webhooks {
+      /**
+       * Constructs and verifies the signature of an Event from the provided details.
+       *
+       * @param {string} payload    Raw text body payload received from Stripe
+       * @param {string} header     Value of the `stripe-signature` header from Stripe
+       * @param {string} secret     Your Webhook Signing Secret
+       * @param {number} tolerance  Seconds of tolerance on timestamps
+       *
+       * @returns Stripe.Event
+       *
+       * @throws Error.StripeSignatureVerificationError
+       */
+      constructEvent(
+        payload: string,
+        header: string,
```

Comment:
`payload` and `header` can be either `string` or `Buffer`s, cf. https://github.com/stripe/stripe-node/blob/f6e5ad36e058fba9aa36fc030f39818b3a77c24f/lib/Webhooks.js#L71-L72

Reply:
Thanks fixed. Note that this required adding `@types/node` as a dependency, meaning it could add a bit of bloat for non-TS users. Given this is a server-only package, I don't think that's the end of the world.